uptane-standard.md

```diff
@@ -648,11 +648,11 @@ The inventory database MUST record the following pieces of information:
 * Per ECU:
     * A unique identifier (such as a serial number)
     * The vehicle identifier the ECU is associated with
-    * A public key
-    * The format of the public key
+    * An ECU key (symmetric or asymmetric; for asymmetric keys only the public part SHOULD be stored)
+    * The format of the ECU key
```

Comment:
What do we mean by format?

Reply:
Pushed a new commit to address that, thanks.